packages/edit-site/src/components/page/style.scss

```diff
@@ -2,7 +2,7 @@
 	color: $gray-800;
 	background: $white;
 	flex-grow: 1;
-	overflow: auto;
+	overflow: hidden;
```

Comment:
Without `overflow: hidden`, the upper left corner of the canvas area would not be rounded.

![canvas-radius](https://github.com/WordPress/gutenberg/assets/54422211/0cf0cd9d-8db9-4211-a379-88e70876fea8)